Src/Common/CrossComponentCorrelationEventSource.cs

```diff
@@ -84,8 +84,12 @@ private string GetApplicationName()
             try
             {
 #if NETCORE
+#if NETSTANDARD1_3
+                name = "Undefined"; // netstandard1.3 does not have API to retrieve Assembly/AppDomain name
```

Comment:
func tests run on netcoreapp1.0 that is netstandard1.5.

The only way to hit this path is to have .net46 framework app with aspnetcore and AI for aspnetcore.
I can create a new app for functional tests and test it, but I'm not sure this weird combination worth effort. Also, it seems nothing bad could happen in production because it's undefined, correct?